dev/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+services:
+
+  db:
+    image: postgres:14
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_MULTIPLE_DATABASES="squid-archive","squid","graph","graph-gamedao"
+    volumes:
+      - ./config/init-db.sh:/docker-entrypoint-initdb.d/init-db.sh
+    ports:
+      - 5432:5432
+
+  ingest:
+    depends_on:
+      - db
+    restart: on-failure
+    image: subsquid/substrate-ingest:firesquid
+    command: [
+       "-e", "wss://node.dev.sub.zero.io",
+       "-c", "5", # allow up to 20 pending requests for the above endpoint (default is 5)
+       "--out", "postgres://postgres:postgres@db:5432/squid-archive"
+    ]
+
+  gateway:
+    depends_on:
+      - db
+    image: subsquid/substrate-gateway:firesquid
+    environment:
+      DATABASE_MAX_CONNECTIONS: 5
+      RUST_LOG: "actix_web=info,actix_server=info"
+    command: [
+       "--database-url", "postgres://postgres:postgres@db:5432/squid-archive",
+    ]
+    ports:
+      - "8888:8000"
+
+  # Explorer service is optional.
+  # It provides rich GraphQL API for querying archived data.
+  # Many developers find it very useful for exploration and debugging.
+  explorer:
+    image: subsquid/substrate-explorer:firesquid
+    environment:
+      DB_TYPE: postgres # set to `cockroach` for Cockroach DB
+      DB_HOST: db
+      DB_PORT: "5432"
+      DB_NAME: "squid-archive"
+      DB_USER: "postgres"
+      DB_PASS: "postgres"
+    ports:
+      - "4444:3000"
+
+  graph-gamedao:
+    image: public.ecr.aws/zero.io/graph-gamedao:v62
+    restart: always
+    environment:
+      IPFS_API_URL: https://ipfs.infura.io:5001/api/v0
+      IPFS_CLIENT_ID: ""
+      IPFS_CLIENT_SECRET: ""
+      SERVER_PORT: 4000
+      CHAIN_RPC_URL: wss://node.dev.sub.zero.io/
+      CHAIN_RPC_URL_OVERRIDE: wss://node.dev.sub.zero.io/
+      DATABASE_URL: postgres://postgres:postgres@db:5432/graph-gamedao?schema=public
+
+  graph:
+    image: hasura/graphql-engine:v2.12.0
```

Comment:
this needs a bump to v2.12.1 as this image does not exist anymore due to security issues (reason for the aws escalation)

Reply:
This one is only used for local development, but I'll also update it. You are right.
I'll update gitlab repo later. And this version change will be one of the changes.